iothub/service/src/IotHubServiceClient.cs

```diff
@@ -134,6 +134,12 @@ public IotHubServiceClient(string hostName, AzureSasCredential credential, IotHu
         /// </summary>
         public ModulesClient Modules { get; protected set; }
 
+        /// <summary>
+        /// Subclient of <see cref="IotHubServiceClient"/> that handles configurations
+        /// getting/adding/setting/deleting configurations.
```

Comment:
"Subclient of <see cref="IotHubServiceClient"/> that handles ~~configurations~~ getting/adding/setting/deleting configurations."

Reply:
I'd also consider adding a link to this primer on automatic device management configurations: https://docs.microsoft.com/azure/iot-hub/iot-hub-automatic-device-management